shared/languages/en.ts

```diff
@@ -1498,4 +1498,5 @@ export const EN = {
    Zoroaster: "Zoroaster",
    Zugspitze: "Zugspitze",
    ZugspitzeDesc: "For each unlocked age, get one point that can be used to provide one extra level to any Great Person that is born from this run",
+   TurnOffFullBuildings: "Shutdown All %{building} With Full Storage"
```

Comment:
I think the wording used in the game is "Turn off" instead of "Shutdown", but please correct me if I am wrong. Also there's conflict in the language file